src/hyppo/dos.asm

```diff
@@ -2349,6 +2351,11 @@ dff_have_next_entry:
 
 dos_opendir:
 
+        ;; assure we are using the sdcard buffer (not the fdc buffer)
+        lda $d689
```

Comment:
Save 3 bytes with:

LDA #$80
TSB $D689

Reply:
Okidoke, will tweak this now.